docs/plugins/snippets/json.lua

```diff
@@ -0,0 +1,388 @@
+--
```

Comment:
Ideally we wouldn't need to redistribute a JSON library and would just have JSON support in the client.